index.js

```diff
@@ -386,7 +386,7 @@
         return x != null && (
           R.type(type.prototype['@@type']) === 'String' ?
             x['@@type'] === type.prototype['@@type'] :
-            R.type(x) === R.nth(1, R.match(/^function (\w*)/, String(type)))
+            R.type(x) === R.nth(1, R.match(/function (\w*)/, String(type)))
```

Comment:
Yes. I have not done this before but I think I will deal with it. I will force-push a new commit instead of this one. Then I will make a new pull request with the right title.